rpcFwManager/RPCMgr.cpp

```diff
@@ -495,7 +494,7 @@ void cmdProcess(std::wstring &processName)
 	if (!processName.empty())
 	{
 		std::wstring msg = L"Enabling RPCFW for process :";
-		msg += processName.c_str();
+		msg += processName;
 		outputMessage(msg.c_str());
```

Comment:
If you are interested, you may consider changing the signature of outputMessage to accept ```const std::wstring&```, as it performs an unnecessary string creation.